### ISSUE TITLE
Ingest "Arabic movie posters" (pudl0100)

### ISSUE THREAD

**Issue by tpendragon** [pulibrary/figgy#1612]
Notes: MODS

Details: https://docs.google.com/spreadsheets/d/1f-c3goE0CC2QJ0Cbk_-JN67nk3fpI9J2Dl37AhGIzos/edit#gid=33
Depends on #1674
Depends on #1716

**Comment by tpendragon** [comment]
Files: https://drive.google.com/drive/u/0/folders/0B4Wo5hgOEFY3ZlZGUU5OS3FIbDA

**Comment by tpendragon** [comment]
http://pudl.princeton.edu/collections/pudl0100

**Comment by jrgriffiniii** [comment]
There are three different filename prefixes for the METS files within the directory `posters`:
- `le1_`
- `eg1_`
- `iq1_`

These seem to be linked to the countries of origin (Lebanon, Egypt, and Iraq), but for Lebanon no `placeTerm` is provided.

Within the directory `lc`, (for "Lobby cards" as opposed to "posters") the files continue for Lebanon, Egypt, and Syria with some additional prefixes:
- `eg0_`
- `eg1_`
- `eg2_`
- `egx_`
- `le0_`
- `le1_`
- `lex_`
- `sy1_`
- `syx_`

**Comment by jrgriffiniii** [comment]
The MODS metadata within the METS files follows this structure:

Field | Element | XPath | Example | Authorities/Encodings
-- | -- | -- | -- | --
Title | titleInfo | mods:mods/mods:titleInfo | The Golden Age of Comedy |  
Resource Type | typeOfResource | mods:mods/mods:typeOfResource | still image |  
Genre | genre | mods:mods/mods:genre | Posters | AAT
Date Created | dateCreated | mods:mods/mods:originInfo/mods:dateCreated | 1901-2000 |  
Language | languageTerm | mods:mods/mods:language/mods:languageTerm | ara | ISO639-2b
Extent | extent | mods:mods/mods:physicalDescription/mods:extent | 1 piece. |  
Collection | collection | mods:mods/mods:relatedItem[@type="host"]/mods:titleInfo/mods:title | Middle Eastern Film Posters Digitization Initiative |  
Identifier | identifier | mods:mods/mods:identifier | le1-0143 |  
Use Rights | accessCondition | mods:mods/mods:accessCondition[@type="useAndReproduction"] | http://www.princeton.edu/~rbsc/research/rights.html |  
Access Restrictions | accessCondition | mods:mods/mods:accessCondition[@type="restrictionOnAccess"] | http://www.princeton.edu/~rbsc/research/rules.html |  
Actor | namePart | mods:mods/mods:name/mods:roleTerm[text()="act"]/../mods:namePart | نعيم حلاوي |  
Director | namePart | mods:mods/mods:name/mods:roleTerm[text()="drt"]/../mods:namePart | شادي حنا |  
Place | place | mods:mods/mods:originInfo/mods:place/mods:placeTerm | Iraq |

**Comment by jrgriffiniii** [comment]
There is additional MODS metadata describing the physical location of these items which might be needed for any MARC records derived from this:
```
   <mods:location>
      <mods:physicalLocation type="text">Princeton University Library. Firestone Library.</mods:physicalLocation>
      <mods:physicalLocation authority="marcorg" type="code">NjP</mods:physicalLocation>
      <mods:holdingSimple>
         <mods:copyInformation>
            <mods:subLocation>Curator's office</mods:subLocation>
            <mods:shelfLocator>Cabinet 11/13</mods:shelfLocator>
         </mods:copyInformation>
      </mods:holdingSimple>
   </mods:location>
```

These values could be mapped to attributes such as `holding_location` or `location`.

**Comment by tpendragon** [comment]
Depends on #1713

**Comment by tpendragon** [comment]
The metadata for all these ingested in prod, they're not generating derivatives.

**Comment by tpendragon** [comment]
Emailed Kim Leaman to get confirmation that the items came over okay.

**Comment by tpendragon** [comment]
Requests/feedback from Kim:

- [ ] Origin data didn't migrate from PUDL (country and year of film, ex. Egypt, 1988)
- [ ] Change location info (All lobby cards should have "ReCAP – Graphic Arts Off-Site Storage, GC191", all posters should have "Mudd Manuscript Library")
- [ ] Add everything to "Selections from the Near East Collections"
- [ ] Batch mark complete

**Comment by escowles** [comment]
These should be ingested as Ephemera, not as ScannedResources

**Comment by tpendragon** [comment]
:|

**Comment by escowles** [comment]
@kelea99 got approval to complete these as ScannedResources instead of reingesting as ephemera.  I've bulk completed them.

**Comment by escowles** [comment]
I was starting to work up a Rake task to validate that METS files were ingested into Figgy, and noticed none of these objects were found because neither the `replaces` nor the ARKs were migrated.  Reopening this ticket to track adding both of those.

**Comment by escowles** [comment]
Closing in favor of #3252
